ci/build_container/build_recipes/gperftools.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-VERSION=2.6.3
+VERSION=2.6.90
```

Comment:
This is a pre-release, right? I would rather not run an RC. Can we wait until the final comes out?

Reply:
It is a proper release but he does mention it is considered an RC for 2.7. Happy to defer on this one.

Reply:
I would prefer that we skip this one until 2.7 comes out. Would prefer to let other people debug memory allocator issues. :)